_posts/2020/2020-12-10-how-to-structure-css-classes.md

```diff
@@ -25,28 +25,28 @@ But if you start to search for methods and techniques to organize your CSS, you
 * [Attribute Modules for CSS (AMCSS)](https://amcss.github.io/){:target="_blank"} _by Glen Maddern & Ben Schwarz, introduced 2014_
 * [Scalable and Modular Architecture for CSS (SMACSS)](http://smacss.com/){:target="_blank"} _by Jonathan Snook, introduced 2015_
 
-The universe of different CSS methodologies is even bigger and grows from time to time, although the list of the most popular ones seems not to change.
+The universe of other CSS methodologies is even bigger and grows from time to time, although the list of the most popular ones seems not to change.
```

Comment:
The universe of CSS methodologies...
(More in the sense of "If I would list ALL methodologies, this list would be much longer... But these are the most important ones..." )